vignettes/howto-dm-theory.Rmd

```diff
@@ -74,11 +74,12 @@ library(tidyverse)
 library(dm)
 library(nycflights13)
 
-flights %>%
+(nycflights13_df <- 
```

Comment:
I'm not a great fan of this syntax, in particular with pipes. Mentioning `nycflights13_df` in a separate line to print it is more explicit.

Reply:
Good point. Changed this to match your suggestion.